### PR TITLE
BiglyBT

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: cleaners/biglybt.xml [hunk 1]
```diff
@@ -29,17 +32,17 @@
...
-    <label>Cache</label>
-    <description>Delete the cache</description>
+    <label>IPFilter Cache</label>
+    <description>Delete the IPFilter Cache</description>
     <action command="delete" search="file" path="~/.biglybt/ipfilter.cache"/>
     <action command="delete" search="file" path="$APPDATA\BiglyBT\ipfilter.cache"/>
   </option>
   <option id="logs">
     <label>Logs</label>
-    <description>Delete the logs</description>
```
Comment (az0) [inline cleaners/biglybt.xml]: "Delete the logs" is already translated in many languages, so it's best to reuse the existing string unless there's an important reason

File: cleaners/biglybt.xml [hunk 2]
```diff
@@ -29,17 +32,17 @@
         <action command="delete" search="glob" path="$APPDATA\BiglyBT\active\*.bak"/>
   </option>
   <option id="cache">
-    <label>Cache</label>
```
Comment (az0) [inline cleaners/biglybt.xml]: These two strings are already translated into many languages

**Comment by Tobias-B-Besemer** [comment]
K, I will fix the regressions...

**Comment by Tobias-B-Besemer** [comment]
OK, solved.

**Comment by az0** [comment]
I'm closing this in favor of https://github.com/az0/cleanerml/pull/150

**Pull request closed without merging**